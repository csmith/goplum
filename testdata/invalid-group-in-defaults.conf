defaults {
  interval = 30s
  timeout = 20s
  groups = ["nonexistent-group"]
}

alert debug.sysout "test" {}

check debug.random "test-check" {
}