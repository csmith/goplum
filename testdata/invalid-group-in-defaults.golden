"error configuring check test-check: no group named 'nonexistent-group'"